packages/astropy/meta.yaml

```diff
@@ -16,6 +19,26 @@ requirements:
     - packaging
     - numpy
     - pyerfa
+    - pyyaml
 test:
   imports:
     - astropy
+    - astropy.config
```

Comment:
So I take it that with these extra imports the import test fails without your patches?

Reply:
I couldn't figure out how to run the tests locally (they don't seem to run when you build the package, which I was able to do), but at least some of them would fail without the patches, in particular `astropy.coordinates` (e.g., do `import astropy.coordinates` in the [current REPL](https://pyodide.org/en/stable/console.html)). I just added all sub-packages to the tests to make sure they all import. I'll add the config one now as well...

Reply:
Incidentally, the way you would run these tests is:
```sh
pytest -k "chrome and astropy" packages/test_packages_common.py
```